setup.py

```diff
@@ -21,7 +21,7 @@
     description="Asynchronous library to control Shelly devices.",
     long_description=README_FILE.read_text(encoding="utf-8"),
     long_description_content_type="text/markdown",
-    packages=find_packages(),
+    packages=find_packages(exclude=["tests", "tests.*"]),
```

Comment:
Should we also exclude the `tools` folder?

Reply:
It was new to me that `find_packages()` would even find the tests as a package, but it makes sense that it is.
However I think if we want to create a package only for `aioshelly` we should just specify it instead of exclusions as it will be more future proof, for example:
https://github.com/home-assistant-libs/aiowebostv/blob/923f8c4705ad8dc9a317f05da3c18f6071cec73d/setup.py#L22

Reply:
I define packages exactly the same way 😄 I think it's worth going in this direction.